hazelcast/include/hazelcast/client/serialization/pimpl/data_output.h

```diff
@@ -349,6 +349,7 @@ template<typename T>
 typename std::enable_if<std::is_same<std::vector<byte>, T>::value ||
                           std::is_same<std::vector<char>, T>::value ||
                           std::is_same<std::vector<bool>, T>::value ||
+                          std::is_same<std::vector<int8_t>, T>::value ||
```

Comment:
Could you add this to the function declaration as well ?

Reply:
Done. Could you please also check the test case as well? Does it make sense to you?

Reply:
Yes, it looks fine. Endianness doesn't matter for `int8_t` but it is better to have it in case some absurd  failure.